contract/i18n/contract.pot

```diff
@@ -2244,3 +2244,75 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:contract.contract_contract_form_view
 msgid "on"
 msgstr ""
+
+#. module: contract
```

Comment:
I had to add these lines in .pot cause without `account_reports ` module it doesn't load months translation cause only in this module are "code" type.
Without it, in `account` base module there're only `model` translation type

Reply:
Thank you so much Denis!
You told me before, but I didn't quite understand what you meant.
How can I check if .pot is correctly generated?
Is a CI/CD operation?

Reply:
> Thank you so much Denis! You told me before, but I didn't quite understand what you meant. How can I check if .pot is correctly generated? Is a CI/CD operation?

You can do it manually:

Settings > Translations > Export > 'New language / PO file'

The generated one should be correct.

Here, it will be generated automatically after merge

Reply:
Thank you again
I found them in .pot

```
#. module: contract
#: code:addons/contract/models/contract_line.py:0
#, python-format
msgid "January"
msgstr ""
```

Reply:
![image](https://github.com/OCA/contract/assets/93990941/f3f23376-af2b-49ac-89bc-b7605ba9db4b)